target_tools/attributes_provider.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2021 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""AttributesProvider"""
+
+from target_python_sdk.utils import is_list
+from target_tools.messages import attribute_not_exist
+from target_tools.constants import REQUEST_TYPES
+
+def create_indexed(response):
+    """
+    :param response: (delivery_api_client.Model.delivery_response.DeliveryResponse)
+        Target View Delivery API response
+    """
+    result = {}
+    for request_type in REQUEST_TYPES:
+        if response[request_type] and response[request_type].mboxes and is_list(response
```

Comment:
Use getattr() here to access request_type from the response, since it's a DeliveryResponse and not a dict